src/core_plugins/console/public/src/directives/history.html

```diff
@@ -9,7 +9,7 @@
       ng-mouseenter="history.viewingReq = req"
       ng-mouseleave="history.viewingReq = history.selectedReq"
       ng-dblclick="history.restore(req)"
-      title="{{ req.description }}"
+      aria-label="{{:: 'Request: ' + history.describeReq(req) }}"
```

Comment:
should this use aria-labelledby? [...If there is visible text labeling the element, use aria-labelledby instead.](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute).